single-node/src/Explicit-Lagrange-Kokkos/SGH_solver/force_sgh.cpp

```diff
@@ -2,275 +2,261 @@
 #include "mesh.h"
 #include "state.h"
 
-
-
-
-
 // -----------------------------------------------------------------------------
 // This function calculates the corner forces and the evolves stress (hypo)
-//------------------------------------------------------------------------------
-void get_force_sgh(const CArrayKokkos <material_t> &material,
-                   const mesh_t &mesh,
-                   const DViewCArrayKokkos <double> &node_coords,
-                   const DViewCArrayKokkos <double> &node_vel,
-                   const DViewCArrayKokkos <double> &elem_den,
-                   const DViewCArrayKokkos <double> &elem_sie,
-                   const DViewCArrayKokkos <double> &elem_pres,
-                   const DViewCArrayKokkos <double> &elem_stress,
-                   const DViewCArrayKokkos <double> &elem_sspd,
-                   const DViewCArrayKokkos <double> &elem_vol,
-                   const DViewCArrayKokkos <double> &elem_div,
-                   const DViewCArrayKokkos <size_t> &elem_mat_id,
-                   DViewCArrayKokkos <double> &corner_force,
-                   const double fuzz,
-                   const double small,
-                   const DViewCArrayKokkos <double> &elem_statev,
-                   const double dt,
-                   const double rk_alpha
-                   ){
-    
-
-    
+// ------------------------------------------------------------------------------
+void
```

Comment:
Can't say I'm a fan of the return type being above the function name; but the democratic process wins on this one.

Reply:
I understand the feeling.  It was something I ran into at kitware (at least for the projects I was on).  It's originally from the GNU style guide. The argument is that having the function name on its own line makes it easier to read because the function names will always be in the same column. This is especially true when there are complex return types. For simple return types, it is less of an issue.But for complex return types it can get messy and difficult to just find the function name.  For example (from Boost, I hope we never get this gross):

```
template <class A1, class A2, class A3, class A4, class A5, class A6>
inline typename normalise<policy<>, A1, A2, A3, A4, A5, A6>::type make_policy(const A1&, const A2&, const A3&, const A4&, const A5&, const A6&)
{ 
   typedef typename normalise<policy<>, A1, A2, A3, A4, A5, A6>::type result_type;
   return result_type(); 
}
```

An added benefit is it gives us a bit more space to write more descriptive function names without extending past column 80, which is not really a requirement anymore, but it is good for code readability on smaller screens (see laptops).  

All that being said, it did take me some time to get used to, but I learned to like it. I am open to switching it back though.  Yay democracy!

Reply:
I like the improvements to readability, formatting, and syntax

Reply:
I'm with Adrian and Daniel, I like the return type in front of the function.

Reply:
Just updated.  Im rebuilding and testing now to make sure nothing broke.